docs/figures/report.txt

```diff
@@ -44,13 +44,13 @@ Impacted length                     : 8 m
 
 Critical flow velocity              : 0.3 m/s
 
-Condition             | Reference file (without measure)       | File with measure
-  750.0 m3/s          | ---                                    | ---     (measure not active)          
- 1300.0 m3/s          | ---                                    | ---     (measure not active)          
- 1700.0 m3/s          | ---                                    | ---     (measure not active)          
- 2100.0 m3/s          | ---                                    | ---     (measure not active)          
- 2500.0 m3/s          | reference-Q1_map.nc                    | measure-Q1_map.nc                     
- 3200.0 m3/s          | reference-Q2_map.nc                    | measure-Q2_map.nc                     
+Condition             | Reference file (without intervention)       | File with intervention
```

Comment:
indenting

Reply:
Unsure what causes this, first I thought the increase in char might cause it, but I tried running the tide cfg and I didnt get the weird indenting, might just be created by hand?
![notepad++_uG8zM1NUDN](https://github.com/user-attachments/assets/37b75195-e104-4628-8f90-b11f06328486)

Reply:
updated in example 1 and 2 txt files by hand also with the expected intendting for comments as I got from tide.